include/real/exact_number.hpp

```diff
@@ -240,10 +550,10 @@ namespace boost {
              */
             bool operator>(const exact_number& other) const {
                 std::vector<int> zero = {0};
-                if (this->digits == zero) {
-                    if (other.digits == zero || other.positive)
-                        return false;
-                    else
+                if (this->digits == zero || this->digits.empty()) {
```

Comment:
When are digits empty?

Reply:
On the default constructor, digits constructs to empty. I suppose instead we could construct to be 0, and get rid of checking for empty

Reply:
I tried changing it but it causes nearly all tests to fail, we can leave that change to later.